attestation-agent/kbs_protocol/src/bin/trustee-agent/README.md

```diff
@@ -0,0 +1,28 @@
+#Trustee agent
+A simple tool to attest and fetch secrets from Trustee
+
+Trustee agent is a part of [confidential-containers](https://github.com/confidential-containers)
+[guest-components](https://github.com/confidential-containers/guest-components)
+project but can be used for confidential VMs as well.
+
+Trustee agent is using attestation-agent's kbs_protocol client and
+attesters to gather hardware-based confidential-computing evidence
+and send it over to Trustee.
+
+A resource with exact same path must be uploaded to Trustee before get_resource is called.
+
+
+##Build:
```

Comment:
```suggestion
## Build:
```

Reply:
Yes, thanks, I fixed it with ## BUILD: ##